modules/admin_manual/pages/configuration/general_topics/search.adoc

```diff
@@ -1,59 +1,280 @@
+----
+
+* xref:configuration/server/occ_command.adoc#onfig-commands[occ Config Commands command set] to configure the app.
++
+Examples:
++
+List app settings::
++
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} config:list search_elastic
+----
++
+[source,plaintext]
+----
+{
+    "apps": {
+        "search_elastic": {
+            "enabled": "yes",
+            "group": "content_searchers",
+            "installed_version": "2.1.0",
+            "nocontent": "false",
+            "scanExternalStorages": "1",
+            "servers": "elastic:xxxxxxx@172.17.0.3:9200",
+            "types": "filesystem"
+        }
+    }
+}
+----
+
+Set app options::
++
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set \
+    search_elastic scanExternalStorages --value 0
+----
++
+or
++
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set \
+    search_elastic scanExternalStorages --value 1
+----
+
+== App Modes
+
+The Full Text Search app provides two modes, which are *active* and *passive*.
+
+=== Active Mode
+
+After enabling the app, it will be default in _active mode_::
++
+* File changes will be indexed in background jobs. +
+System cron is recommended, otherwise a lot of jobs might queue up.
+* Search results will be based on Elasticsearch.
+* Search functionality based on ownCloud core database queries will no longer be used.
++
+NOTE: Active mode can cause a downtime for search when start indexing in an already heavily used instance, because it takes a while until all files have been indexed.
+
+=== Passive Mode
+
+To do an initial full indexing without the app interfering it, it can be put in _passive_ mode::
++
+* The administrator can run occ commands changing the search configuration without notice to the users.
+* The app will not index any changes by itself.
+* Search results will still be based on ownCloud core database queries.
+
+=== Changing the App Mode
+
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set \
+    search_elastic mode --value passive
+----
+
+or
+
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set \
+    search_elastic mode --value active
+----
+
+== Restrict Search Results
+
+=== Index Metadata Only
+
+If you only want to use the Full Text Search app as a more scalable search on filenames, you can disable content indexing by setting the option `nocontent` to `true`, which defaults to `false`:
+
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set \
+    search_elastic nocontent --value true
+----
+
+[NOTE]
+====
+* You have to reindex all files if you change this back to `false`. Setting it to `true` does not require reindexing.
+* It may be a maybe a more flexible way to go with limiting full text search to certain groups by setting the option `group.nocontent`, see xref:limit-metadata-search-for-groups[below] for details.
```

Comment:
```suggestion
* It may be a more flexible way to go with limiting full text search to certain groups by setting the option `group.nocontent`, see xref:limit-metadata-search-for-groups[below] for details.
```